src/nifake/metadata/attributes_addon.py

```diff
@@ -6,4 +6,12 @@
 attributes_codegen_method = {
 }
 
+attributes_converters = {
+    1000007: { 'python_api_converter_name_to_driver': 'convert_timedelta_to_seconds', 
```

Comment:
We could store this as "attribute_class". At add_all time, we use the regular attributes (such as AttributeViInt32), but here you can overload to specify the attribute class you need (such as AttributeTimeDeltaToSeconds)

Reply:
Switched to additional attribute classes